packages/react/src/metadata.tsx

```diff
@@ -29,6 +29,32 @@ const FieldMetadataFragment = graphql(/* GraphQL */ `
     configuration {
       __typename
       fieldType
+      validations {
+        ... on GadgetRegexFieldValidation {
+          __typename
```

Comment:
You can stick this outside the `... on` parts before the first one to select it always, regardless of which union element the server returns